### PR TITLE
[DEN-6] WAF - Gets All Custom Rule Sets implementation

### PR DESCRIPTION
**Pull request by ssaluja4**
Integrates the following WAF API (Get All Custom Rule Sets) into GO Lang SDK

https://dev.vdms.com/cdn/api/index.html#Media_Management/Web-Security/Get-All-Custom-Rule-Sets.htm%3FTocPath%3DWeb%2520Application%2520Firewall%7C_____11

### CODE REVIEW

File: edgecast/waf/custom_rule.go [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+	//Indicates the name of the custom rule set.
+	Name string `json:"name"`
+}
+
+// Used to handle value returned for LastModifiedDate to allow for implementation of UnmarshalJSON below
+type shortDateTime struct {
+	string
+}
+
+//Allows for LastModifiedDate field within customRuleSets struct to be of formatted datetime string
```
Comment (stevenpaz) [inline edgecast/waf/custom_rule.go:30]: shortDateTime struct and UnmarshalJSON funcs should be removed, they will conflict with Frank's code

**Review by stevenpaz — CHANGES_REQUESTED** [review]
Just need to remove the duplicated code from Frank's branch, otherwise looks good

**Comment by ssaluja4** [comment]
@stevenpaz updated based on your comments. Please review. Thanks.

**Comment by stevenpaz** [comment]
Looks good to me

**Review by stevenpaz — APPROVED** [review]

**Pull request merged**